### DISCUSSION TITLE
Cannot create pull request

### DISCUSSION THREAD

**Discussion by JohnKara** [beetbox/beets#4279] [Q&A] [answered]
I'm trying to create a new pull request from pycharm and when I press 'Create Pull Request' button it gives me this error:

Can't create a new pull request: Although you appear to have the correct authorization credentials, the `beetbox` organization has enabled OAuth App access restrictions, meaning that data access to third-parties is limited. For more information on these restrictions, including how to enable this app, visit https://docs.github.com/articles/restricting-access-to-your-organization-s-data/

Does anyone know how can I fix this error?

**Comment by sampsyo** [comment, marked as answer]
Interesting! I don't know much (anything) about PyCharm, but is it trying to push to the `beetbox/beets` repository? Because you don't have write access to that repo, you'll want to first create a fork (or rather, use the one you [already have](https://github.com/JohnKara/beets)) and push your changes there before creating a pull request.

**Reply by JohnKara** [reply]
I have already pushed my changes to my forked repo. I think the problem is that beetbox organization doesn't trust Jetbrains' products (one of them is PyCharm) in OAuth App access restrictions.

**Reply by sampsyo** [reply]
Ah… I guess I'm just surprised that API access is required to open a PR. Did you know it's also pretty easy to do in the web UI—in fact, this link will do it?
https://github.com/beetbox/beets/compare/master...JohnKara:master

**Reply by JohnKara** [reply]
No I've never tried that. Thank you! 👍
